collect_app/src/test/java/org/odk/collect/android/configure/SettingsImporterTest.java

```diff
@@ -45,6 +50,10 @@
 
     @Before
     public void setup() {
+        getComponent(ApplicationProvider.<Collect>getApplicationContext()).projectImporter().importDemoProject();
```

Comment:
Instead of setting up a current project, each test could just pass in a random project ID to `getGeneralSettings` etc and `fromJSON`. There's no need to actually have the project setup, just the same id used for the import and assertions. This is another place where a test version of `SettingsProvider` would be useful.